src/store/index.ts

```diff
@@ -243,36 +243,36 @@ const resolvers: Resolvers = {
         },
         login: async (_root, { username, password }: MutationRegisterArgs, { cache, communityPromise }): Promise<User | undefined> => {
             await communityPromise
-            let [success, ambientUser] = await appUser.login(username!, password!)
-            if (!success || !ambientUser) {
+            let [success, user] = await appUser.login(username!, password!)
+            if (!success || !user) {
                 return undefined
             }
 
-            await ambientUser.load()
-            let user = {
-                did: ambientUser.did!,
+            await user.load()
+            let userObj = {
+                did: user.did!,
                 namespace: AppUser.userNamespace?.toString()!,
-                username: ambientUser.userName,
+                username: user.userName,
                 loggedIn: true,
             }
 
             cache.writeQuery({
                 query: CURRENT_USER,
-                data: { me: user }
+                data: { me: userObj }
             })
 
-            return user
+            return userObj
```

Comment:
this is fine - but I was trying to migrate to the graphql objects being *the* thing... so instead of "user" being the appUser "user" would be the graphql user... because I don't think we need that appUser stuff (in the future).

That said - this is totally fine for where we're at :).